packages/webdriverio/src/commands/element/getCSSProperty.ts

```diff
@@ -81,13 +103,13 @@ export default async function getCSSProperty (
      * otherwise expand it and run the command for each longhand property.
      */
     if (!cssShorthandProps.isShorthand(cssProperty)) {
-        const cssValue = await this.getElementCSSValue(this.elementId, cssProperty)
+        const cssValue = await this.getElementCSSValue(this.elementId, cssProperty, pseudoElement)
```

Comment:
The WebDriver spec afaik doesn't allow to fetch for pseudo elements. A way to work around this would be to check whether this option is given and if so, call `execute` and fetch the css property through running the same script we have in the devtools package.

Reply:
So something like this?

```ts
const cssValue = this.execute(`window.getComputedStyle(${this.elementId}, ${pseudoElement})[${cssProperty}]`)
return parseCSS(cssValue, cssProperty)

```

Reply:
no, this works:

```js
const cssValue = this.execute(
    (elem, pseudoElement, cssProperty) => window.getComputedStyle(elem, pseudoElement)[cssProperty],
    element,
    pseudoElement,
    cssProperty
)
return parseCSS(cssValue, cssProperty)
```